### PR TITLE
test(hx-checkbox): implement component tests

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Test Case `<hx-checkbox>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2000

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-checkbox/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                const id = 'hxCheckbox';
+                const component = /** @type { HXCheckboxElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxTick with checkbox type <hx-icon>', async () => {
```
Comment (lalithkarikelli) [inline src/elements/hx-checkbox/index.spec.js:64]: Please change it to

`it(' markup should contain a #hxTick with checkmark type <hx-icon>')`

File: src/elements/hx-checkbox/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                const iconType = 'checkmark';
+                const component = /** @type { HXCheckboxElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryType = query.type;
+
+                expect(queryType).to.equal(iconType);
+            });
+
+            it('markup should contain a #hxMinus with checkbox type <hx-icon>', async () => {
```
Comment (lalithkarikelli) [inline src/elements/hx-checkbox/index.spec.js:75]: Please change it to 

` it('markup should contain a #hxMinus with minus type <hx-icon>')`

File: src/elements/hx-checkbox/index.spec.js (hunk 1 above)
Comment (BY00565233) [reply src/elements/hx-checkbox/index.spec.js]: done

File: src/elements/hx-checkbox/index.spec.js (hunk 2 above)
Comment (BY00565233) [reply src/elements/hx-checkbox/index.spec.js]: done

**Pull request merged**